src/docs/resources/videos.md

```diff
@@ -23,14 +23,21 @@ Coding mistakes, solutions, and snazzy intro music included.
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/yr8F2S3Amas?rel=0" frameborder="1" allow="autoplay; encrypted-media" allowfullscreen></iframe>
 Episode One: The Boring Flutter Show<br>
-[The Boring Flutter Show playlist](https://www.youtube.com/watch?v=yr8F2S3Amas&list=PLOU2XLYxmsIK0r_D-zWcmJ1plIcDNnRkK)
+[The Boring Flutter Show playlist](https://www.youtube.com/playlist?list=PLOU2XLYxmsIK0r_D-zWcmJ1plIcDNnRkK)
 
 ### Flutter Widget of the Week
 
 Do you have 60 seconds? Each one-minute video highlights a Flutter widget.
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/lkF0TQJO0bA?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
-[Flutter Widget of the Week playlist](https://www.youtube.com/watch?v=lkF0TQJO0bA&list=PLOU2XLYxmsIL0pH0zWe_ZOHgGhZ7UasUE)
+[Flutter Widget of the Week playlist](https://www.youtube.com/playlist?list=PLOU2XLYxmsIL0pH0zWe_ZOHgGhZ7UasUE)
+
+### Flutter Widgets 101
+
+Five-to-ten minute videos on how to use Flutter widgets.
```

Comment:
I might give this a little more info:

"Five-to-ten minute videos on StatelessWidget, StatefulWidget, and other building block widgets that every Flutter developer needs to know about."